app/main.py

```diff
@@ -1,3 +1,58 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> float:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance" or float) -> "Distance":
```

Comment:
1. Add import and remove quotes in all places
`from __future__ import annotations`
2. It also accepts int
3. Not need to use or.. Use Union[comma separated types] or | separed(int | float | str)